doc/README.md

```diff
@@ -0,0 +1,60 @@
+# Documentation
+
+This directory contains mostly reference documentation for the Luanti engine.
+For a less prescriptive and more guiding documentation, also look at:
+https://docs.luanti.org
+
+Note that the inner workings of the engine are not well documented. It's most
+often better to read the code.
+
+Markdown files are written in a way that they can also be read in plain text.
+When modifying, please keep it that way!
+
+Here is a list with descriptions of relevant files:
+
+## Server Modding
+
+- [lua_api.md](lua_api.md): Server Modding API reference. (Not only the Lua part,
```

Comment:
It's a bit of a shame that we have to write `[x.md](x.md)` but I just tested and there seems to be no nice alternative. `<x.md>` does not work :cry: